src/main/mondrian/olap/Mondrian.xml

```diff
@@ -1844,6 +1854,19 @@ Revision is $Id$
 
     <Element type="AggFactCount" class= "AggColumnName"/>
 
+    <Element type="AggMeasureFactCount" class= "AggColumnName">
+        <Attribute name="factColumn" required="false">
```

Comment:
Why is factColumn not required? Are there scenarios where we don't set it? I also don't see any tests where it's not set.